x/merkledb/trie_test.go

```diff
@@ -32,6 +46,10 @@ func getNodeValueWithBranchFactor(t ReadOnlyTrie, key string, bf BranchFactor) (
 		if err != nil {
 			return nil, err
 		}
+		if len(nodePath) == 0 {
```

Comment:
The comment on `getPathTo` says `// Always returns at least the root node.` so this should never happen, right?